frontend/src/components/NavBar/index.jsx

```diff
@@ -66,28 +76,35 @@ function NavBar() {
                     Content Creator Tool
                 </Link>
-                <div className="w-full h-[5%] flex items-center mt-auto text-lg justify-evenly cursor-pointer" 
-                    onClick={()=>{changeTheme(theme === "dark" ? "light":"dark");console.log(theme)}}>
+                <button className="w-full h-[5%] flex items-center mt-auto text-lg justify-evenly cursor-pointer" onClick={handleLogout}>
+                    Logout
+                </button>
+
+                <button className="w-full h-[5%] flex items-center mt-auto text-lg justify-evenly cursor-pointer" onClick={() => changeTheme(theme === "dark" ? "light" : "dark")}>
                     <div className="h-full w-[25px] relative flex items-center">
-                        <AnimatePresence>
-                            {theme === "light" && <motion.div className="absolute"
-                                                    initial={{y:-10,opacity:0,rotate:90}}
-                                                    animate={{y:0,opacity:1,rotate:0}}
-                                                    exit={{y:-10,opacity:0,rotate:90}}
-                                                ><MdOutlineWbSunny className="h-full w-[25px]"/></motion.div>
-                }
-                        </AnimatePresence>
-                        <AnimatePresence>
-                            {theme === "dark" && <motion.div className="absolute"
-                                                    initial={{y:10,opacity:0,rotate:-90}}
-                                                    animate={{y:0,opacity:1,rotate:0}}
-                                                    exit={{y:10,opacity:0,rotate:-90}}
-                                                ><LuMoon className="h-full w-[25px]"/></motion.div>
-                }
-                        </AnimatePresence>
+                        {theme === "light" && (
+                            <motion.div
+                                className="absolute"
+                                initial={{ y: -10, opacity: 0, rotate: 90 }}
+                                animate={{ y: 0, opacity: 1, rotate: 0 }}
+                                exit={{ y: -10, opacity: 0, rotate: 90 }}
+                            >
+                                <MdOutlineWbSunny className="h-full w-[25px]" />
+                            </motion.div>
+                        )}
+                        {theme === "dark" && (
+                            <motion.div
+                                className="absolute"
+                                initial={{ y: 10, opacity: 0, rotate: -90 }}
+                                animate={{ y: 0, opacity: 1, rotate: 0 }}
+                                exit={{ y: 10, opacity: 0, rotate: -90 }}
+                            >
+                                <LuMoon className="h-full w-[25px]" />
+                            </motion.div>
+                        )}
```

Comment:
Automatic code design made changes here. If we have not agreed on the usage of a specific tool, then be careful to avoid things like these. They populate reviews.

Reply:
This had to be changed to fix a sonarcloud issue.